pyseir/inference/model_fitter.py

```diff
@@ -459,23 +459,23 @@ def _fit_seir(
         l = locals()
         model_kwargs = {k: l[k] for k in self.model_fit_keys}
 
-        # Last data point used in Fit
-        last_data_point_used = t0 + t_break + 14 + t_delta_phases + 14
+        # Last data point in ramp 2
+        last_data_point_ramp_2 = t0 + t_break + 14 + t_delta_phases + 14
         # Number of future days used in second ramp period
-        number_of_future_days_used = last_data_point_used - self.days_since_ref_date
+        number_of_future_days_used = last_data_point_ramp_2 - self.days_since_ref_date
         # Multiplicative chi2 penalty if future_days are used in second ramp period (set to 1 by default)
         future_days_penalty = 1.0
```

Comment:
Do we think this needs to be higher? Right now it is of order 1, which is very small relative to overall chi2.  Might make this 5 or so.